client/src/main/resources/application.conf

```diff
@@ -8,11 +8,10 @@ grasshopper {
   client {
     parser {
       host = "localhost"
+      host = ${?GRASSHOPPER_PARSER_HOST}
```

Comment:
It uses the first `host = "localhost"` as the default, but _if_ there's an `GRASSHOPPER_PARSER_HOST` envvar set, it'll use that value instead.